apps/hasura/metadata/databases/masterbots/tables/public_user.yaml

```
@@ -167,4 +129,4 @@ update_permissions:
         user_id:
           _eq: X-Hasura-User-Id
       check: null
-    comment: ""
+    comment: ""
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)